### PR TITLE
#103 add content files relations

### PR DESCRIPTION
**Pull request by michalstrzelecki**

### CODE REVIEW

File: src/Gzero/Cms/Http/Controllers/Api/ContentFileController.php [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    protected $repository;
+
+    /** @var ContentValidator */
+    protected $validator;
+
+    /** @var Request */
+    protected $request;
+
+    /**
+     * BlockController constructor.
```
Comment (PSkierniewski) [inline src/Gzero/Cms/Http/Controllers/Api/ContentFileController.php]: Wrong controller name

File: src/Gzero/Cms/Http/Controllers/Api/ContentFileController.php [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+use Gzero\Core\Models\File;
+use Gzero\Core\UrlParamsProcessor;
+use Illuminate\Http\Request;
+
+/**
+ * Class ContentFileController
+ *
+ * @SWG\Tag(
+ *   name="content",
+ *   description="Everything about app content"
```
Comment (PSkierniewski) [inline src/Gzero/Cms/Http/Controllers/Api/ContentFileController.php]: This swagger tag already exists, so there will be an error during docs generation. Please make sure that swagger docs are generating with your new methods.

Comment (michalstrzelecki) [reply src/Gzero/Cms/Http/Controllers/Api/ContentFileController.php]: There is no error, I checked it. We can leave it as that or remove this doc block completely like it is done in `ContentTranslationController`.

File: src/Gzero/Cms/Http/Controllers/Api/ContentFileController.php (hunk 1 above)
Comment (michalstrzelecki) [reply src/Gzero/Cms/Http/Controllers/Api/ContentFileController.php]: Changed.

File: tests/functional/api/ContentFileCest.php [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        $file = dispatch_now(CreateFile::image($image, 'Image', $language, $author, [
+            'info'        => 'info text',
+            'description' => 'My image',
+            'is_active'   => true,
+        ]));
+        $translation = dispatch_now(new AddFileTranslation($file, 'New translation', $language, $author,
+            ['description' => 'Description']
+        ));
+
+        $I->sendPUT(apiUrl("contents/$content->id/files"), [
```
Comment (PSkierniewski) [inline tests/functional/api/ContentFileCest.php]: Please add `weight` to check if it works with API since files will be sorted and send with weights

Comment (michalstrzelecki) [reply tests/functional/api/ContentFileCest.php]: added

**Pull request merged**